### PR TITLE
add new fields on Mental Health form

### PR DESCRIPTION
**Pull request by Dutervil**
@cioan @mseaton  @lnball 
have a look that PR
I have created this branch to work on five tickets because of concerning the same form

### CODE REVIEW

**Review by cioan — APPROVED** [review]
Looks good, @Dutervil ! Nice job!

File: configuration/pih/htmlforms/mentalHealth.xml [hunk 1]
```diff
@@ -739,6 +738,63 @@
...
+
+            </div>
+        </div>
+        <div class="section-container" id="radio_button_container">
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.muscleTone"/>
+                    </label>
+                    <obs conceptId="PIH:15034" style="radio" answerConceptIds="PIH:YES,PIH:NO" answerCodes="yes,no" answerSeparator=""/>
```
Comment (lnball) [inline configuration/pih/htmlforms/mentalHealth.xml:792]: I think the answers should be normal/abnormal, not yes/no.  But it's your choice.

**Review by lnball — CHANGES_REQUESTED** [review]
@Dutervil see my comments.

File: configuration/messageproperties/messages_en.properties [hunk 2]
```diff
@@ -1529,13 +1529,20 @@ pihcore.mealsPerDay=Meals per day
...
-pihcore.mentalhealth.cognitive=Cognitive function wnl
+pihcore.mentalhealth.status=Status
+pihcore.mentalhealth.generalAppearance =General appearance within normal limits
+pihcore.mentalhealth.speech=Speech within normal limits
+pihcore.mentalhealth.cognitive=Cognitive function within normal limits
 pihcore.mentalhealth.mood=Mood disorder
 pihcore.mentalhealth.muscleTone=Muscle tone and strength
 
+pihcore.mentalhealth.PoorIntrospection =Poor introspection
+pihcore.mentalhealth.ThoughtContent=Thought content wnl
```
Comment (lnball) [inline configuration/messageproperties/messages_en.properties:1540]: No "wnl".  Please change.

File: configuration/messageproperties/messages_en.properties [hunk 3]
```diff
@@ -1529,13 +1529,20 @@ pihcore.mealsPerDay=Meals per day
...
-pihcore.mentalhealth.speech=Speech wnl
-pihcore.mentalhealth.cognitive=Cognitive function wnl
+pihcore.mentalhealth.status=Status
+pihcore.mentalhealth.generalAppearance =General appearance within normal limits
+pihcore.mentalhealth.speech=Speech within normal limits
+pihcore.mentalhealth.cognitive=Cognitive function within normal limits
 pihcore.mentalhealth.mood=Mood disorder
 pihcore.mentalhealth.muscleTone=Muscle tone and strength
 
+pihcore.mentalhealth.PoorIntrospection =Poor introspection
```
Comment (lnball) [inline configuration/messageproperties/messages_en.properties]: Remove space.

I recommend changing the text to "Introspection" with yes/no or normal/abnormal.  That matches the concept.

File: configuration/messageproperties/messages_en.properties [hunk 4]
```diff
@@ -1529,13 +1529,20 @@ pihcore.mealsPerDay=Meals per day
 pihcore.numberPerDay=Number per day
 zl.socioEconomic.monthsLivingInHaiti=Number of months residing in Haiti
 
-
-pihcore.mentalhealth.generalAppearance =General appearance wnl
-pihcore.mentalhealth.speech=Speech wnl
-pihcore.mentalhealth.cognitive=Cognitive function wnl
+pihcore.mentalhealth.status=Status
```
Comment (lnball) [inline configuration/messageproperties/messages_en.properties:1532]: You do not need this.  Use "pihcore.status"

**Review by lnball — CHANGES_REQUESTED** [review]
More changes...

File: configuration/messageproperties/messages_en.properties (hunk 4 above)
Comment (Dutervil) [reply configuration/messageproperties/messages_en.properties:1532]: @lnball I made the changes asked

Comment (lnball) [reply configuration/messageproperties/messages_en.properties:1532]: If you aren't using this now, can you remove this line?

Comment (Dutervil) [reply configuration/messageproperties/messages_en.properties]: @lnball I remove pihcore.mentalhealth.status

**Review by lnball — APPROVED** [review]
LGTM (looks good to me).  Thanks for making all the little changes @Dutervil

**Comment by Dutervil** [comment]
You are welcome and thank you for assisting me in @lnball

**Pull request merged**